src/pipeline_factory.cpp

```diff
@@ -57,4 +58,16 @@ Status PipelineFactory::create(std::unique_ptr<Pipeline>& pipeline,
     lock.unlock();
     return definition.create(pipeline, request, response, manager);
 }
+void PipelineFactory::revalidatePipelines(ModelManager& manager) {
+    for (auto& [name, d] : definitions) {
```

Comment:
Don't use one letter variable names, it would be better to just define `definition`